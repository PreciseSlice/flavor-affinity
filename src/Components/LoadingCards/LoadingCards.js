import React from 'react';
import fruitSwirl from '../../Assets/fruit-swirl.gif';

const LoadingCards = () => {
  return (
    <div className="card-container">
      <div className="loading-container">
        <h1>loading ingredients...</h1>
        <img src={fruitSwirl} alt="swirling fruit" />
      </div>
    </div>
  );
};

export default LoadingCards;
