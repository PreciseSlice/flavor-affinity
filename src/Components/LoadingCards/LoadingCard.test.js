/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import LoadingCards from './LoadingCards';
import { shallow } from 'enzyme';

describe('LoadingCards', () => {
  let wrapper;
  const mockFn = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<LoadingCards />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});