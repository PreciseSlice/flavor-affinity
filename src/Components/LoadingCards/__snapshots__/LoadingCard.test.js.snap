// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingCards matches snapshot 1`] = `
<div
  className="card-container"
>
  <div
    className="loading-container"
  >
    <h1>
      loading ingredients...
    </h1>
    <img
      alt="swirling fruit"
      src="fruit-swirl.gif"
    />
  </div>
</div>
`;
